casanovo/denovo/model.py

```diff
@@ -1076,7 +1076,7 @@ def _aa_pep_score(
     peptide_score : float
         The peptide score.
     """
-    peptide_score = np.mean(aa_scores)
+    peptide_score = np.exp(np.mean(np.log(aa_scores)))
```

Comment:
I'm not sure if it's strictly necessary to do this calculation in log space for the sequence lengths we'll be dealing with, but I implemented it this way just to reduce any possible rounding errors.